### PR TITLE
Feature: add activate-launchplan command to pyflyte

### PR DESCRIPTION
**Pull request by wckdman**
# TL;DR
Adds a new command to **pyflyte** which simplifies the process of activating new launch plans 

## Type
 - [ ] Bug Fix
 - [X] Feature
 - [ ] Plugin

## Are all requirements met?

 - [X] Code completed
 - [X] Smoke tested
 - [X] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
I found that `SynchronousFlyteClient` has [update_launch_plan](https://github.com/flyteorg/flytekit/blob/master/flytekit/clients/friendly.py#L495) method. So I decided to wrap it up as a CLI-command.

## Tracking Issue
https://github.com/flyteorg/flyte/issues/3587

## Follow-up issue
_NA_

### CODE REVIEW

File: flytekit/clis/sdk_in_container/activate_launchplan.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+_activate_launchplan_help = """
+The activate-launchplan command activates a specified or the latest version of the launchplan and disables a previous version.
+
+- ``launchplan`` refers to the name of the Launchplan
+- ``launchplan_version`` is optional and should be a valid version for a Launchplan version. If not specified the latest will be used.
+"""
+
+
+@click.command("activate-launchplan", help=_activate_launchplan_help)
```
[the remark this answers is not in the document]
Comment (wckdman) [reply flytekit/clis/sdk_in_container/activate_launchplan.py:14]: Totally agree!

Shall I set a default value for this action (`activate/deactivate`) or make it required? WDYT?

[the remark this answers is not in the document]
Comment (pingsutw) [reply flytekit/clis/sdk_in_container/activate_launchplan.py:14]: make it required i think

**Review by pingsutw — COMMENTED** [review]
LGTM, could you merge master into this branch to resolve the conflict

**Comment by wckdman** [comment]
@pingsutw done ✅

**Review by pingsutw — APPROVED** [review]

**Review by kumare3 — APPROVED** [review]
Windows build is failing

**Comment by wckdman** [comment]
@kumare3 it looks like a recurring problem according to "actions" tab. even the build at **master** branch has failed.

**Comment by pingsutw** [comment]
Test is failing in all the PRs for some reason, I'm fixing it today.

**Review by pingsutw — APPROVED** [review]
